packages/host/app/commands/update-skill-activation.ts

```diff
@@ -0,0 +1,52 @@
+import { service } from '@ember/service';
+
+import type * as BaseCommandModule from 'https://cardstack.com/base/command';
+
+import HostBaseCommand from '../lib/host-base-command';
+
+import { SKILLS_STATE_EVENT_TYPE } from '../services/matrix-service';
+
+import type MatrixService from '../services/matrix-service';
+
+export default class UpdateSkillActivationCommand extends HostBaseCommand<
+  BaseCommandModule.UpdateSkillActivationInput,
+  undefined
+> {
+  @service private declare matrixService: MatrixService;
+
+  async getInputType() {
+    let commandModule = await this.loadCommandModule();
+    const { UpdateSkillActivationInput } = commandModule;
+    return UpdateSkillActivationInput;
+  }
+
+  protected async run(
+    input: BaseCommandModule.UpdateSkillActivationInput,
+  ): Promise<undefined> {
+    let { matrixService } = this;
+    let { roomId, skillEventId, value } = input;
+    await matrixService.updateStateEvent(
+      roomId,
+      SKILLS_STATE_EVENT_TYPE,
+      '',
+      async (currentSkillsConfig) => {
+        let newSkillsConfig = {
+          enabledEventIds: [...(currentSkillsConfig.enabledEventIds || [])],
+          disabledEventIds: [...(currentSkillsConfig.disabledEventIds || [])],
+        };
+        if (value) {
```

Comment:
I liked `isActive` as a name for this more, maybe there’s an interface reason for it to be `value` now but it’s less explanatory.

Reply:
👍  I'll adjust it in my next PR